examples/playground/panel-lite-editor.js

```diff
@@ -0,0 +1,131 @@
+const appContainer = "app-container"
+const codeEditor = "code-editor"
+
+const welcomeExample=`\
+import panel as pn
+pn.extension()
+
+pn.pane.Markdown("Welcome! Try clicking the \`run\` button.").servable()
+`
+
+const buttonExample=`\
```

Comment:
## Todo

- [ ] Often two buttons are rendered. Only one is working. I believe its because the Pyodide code is run so fast that it renders before the Bokeh prerendering has finished.